app/src/main/java/com/yagni/view/LoginGUI.java

```diff
@@ -112,14 +198,44 @@ public class LoginGUI implements ActionListener {
         frame.setResizable(false); // disables the resize features of the frame to the user
 
         // add header, body and footer panels to the frame
+>>>>>>> main
         frame.add(headerPanel, BorderLayout.NORTH);
         frame.add(bodyPanel, BorderLayout.CENTER);
         frame.add(footerPanel, BorderLayout.SOUTH);
 
+<<<<<<< HEAD
+        // display all content to the GUI 
+        frame.setVisible(true);
+    }
+
+	//Button click method 
+	@Override
+	public void actionPerformed(ActionEvent e) {
+			String user = userText.getText();
+			String password = passwordText.getText();
+			
+			DbConnection linkDB = new DbConnection(user,password);
+			System.out.println(linkDB);
+			success.setText("");
+			
+		// if the login button is clicked and the user name and password are correct 
+        // then open the EmployeeGUI; display login error otherwise. 
+		if (e.getSource() == button && linkDB.getConnection() != null) {
+			System.out.println("Login Successful!");
+			System.out.println("---------------------------------------");
+			frame.dispose();
+			new EmployeeGUI(linkDB); 
+		
+		} else {
+			success.setText("Invalid credentials! Please Try Again. ");
+			success.setForeground(Color.RED); 	
+		}
+=======
         // display all content to the GUI
         frame.setVisible(true);
 
     }
+>>>>>>> main
```

Comment:
Looks like you have some incoming changes from Head and your working branch that are conflicting. Various points in this whole file are affected.

Reply:
It said I resolved all them locally but I'll go through it again w/ you to be certain.